web-module/src/main/webapp/templates/celStruct/table/col_delete.vm

```diff
@@ -5,6 +5,14 @@
   #set($xredirect = $services.url.getURL())
 #end
 #set($xredirect = $escapetool.url($xredirect))
-#set($url = $services.url.getURL($rowdoc.documentReference, 'delete', "xredirect=${xredirect}"))
+#set($query = "xredirect=${xredirect}")
+#if("$!xcancel" == '')
+  #set($xcancel = "$!request.xcancel")
+#end
+#if($xcancel &&"$!xcancel" !='')
```

Comment:
```suggestion
#if("$!xcancel" !='')
```
If `$xcancel` is null `"$!xcancel"` results in an empty string and leads already to falsing the expression.